icu4c/source/common/dictbe.cpp

```diff
@@ -230,6 +231,7 @@ ThaiBreakEngine::divideUpDictionaryRange( UText *text,
                                                 int32_t rangeStart,
                                                 int32_t rangeEnd,
                                                 UVector32 &foundBreaks,
+                                                UBool isPhraseBreaking,
```

Comment:
If you got compiler error, change to 
                             UBool /* isPhraseBreaking */,